src/Tizen.Applications.EventManager/Tizen.Applications/SystemEvents.cs

```diff
@@ -0,0 +1,898 @@
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public const string StatusValueDisconnected = "disconnected";
+
+        /// <summary>
+        /// Value of charing battery charger
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public const string StatusValueCharging = "charging";
+
+        /// <summary>
+        /// Value of discharging battery charger
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public const string StatusValueDischarging = "discharging";
+    }
+
+    /// <summary>
+    /// Class for system-event of battery level status
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public static class BatteryLevelStatus
+    {
+        /// <summary>
+        /// Eventname for system-event of battery level status
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public const string EventName = "tizen.system.event.battery_level_status";
+
+        /// <summary>
+        /// Key for system-event of battery level status
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public const string StatusKey = "battery_level_status";
+
+        /// <summary>
+        /// Value of empty battery level status
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public const string StatusValueEmpty = "empty";
+
+        /// <summary>
+        /// Value of critical battery level status
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public const string StatusValueCritical = "critical";
+
+        /// <summary>
+        /// Value of low battery level status
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public const string StatusValueLow = "low";
+
+        /// <summary>
+        /// Value of high battery level status
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public const string StatusValueHigh = "high";
+
+        /// <summary>
+        /// Value of full battery level status
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public const string StatusValueFull = "full";
+    }
+
+    /// <summary>
+    /// Class for system-event of usb status
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public static class UsbStatus
+    {
+        /// <summary>
+        /// Eventname for system-event of usb status
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public const string EventName = "tizen.system.event.usb_status";
+
+        /// <summary>
+        /// Key for system-event of usb status
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public const string StatusKey = "usb_status";
+
+        /// <summary>
+        /// Value of disconnected usb status
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public const string StatusValueDisconnected = "disconnected";
```

Comment:
Why don't you provide as enum type?

Reply:
"disconnected" is literally necessary. 
Please refer to native API
https://developer.tizen.org/development/api-references/native-application?redirect=https://developer.tizen.org/dev-guide/5.0.0/org.tizen.native.mobile.apireference/group__CAPI__EVENT__MODULE.html